### DISCUSSION TITLE
ELN Consortium meeting 7

### DISCUSSION THREAD

**Discussion by NicolasCARPi** [TheELNConsortium/TheELNFileFormat#83] [General]
Scheduled on the 14th of November 2024 at 3PM CEST.

https://zoom.us/j/97692980468?pwd=GJaIVTumFrMkv9UykQlA85hMjPqUxT.1

**Comment by NicolasCARPi** [comment]
Hey all,

I won't be able to make it (nor anyone from Deltablot) on this date. Would it be okay to move it to Thursday 14th?

**Reply by FlorianRhiem** [reply]
I'd have to leave at around 4:15 on that day, but perhaps starting at 3:00 or 3:30 could work?

**Reply by NicolasCARPi** [reply]
@FlorianRhiem Yeah, we could start earlier, no problem for me.

**Comment by cbarillari** [comment]
Hi @NicolasCARPi , how can one join this meeting?

**Reply by NicolasCARPi** [reply]
@cbarillari Click the zoom link from the first message (I've just added it!) ;)

**Reply by cbarillari** [reply]
Thank you!

**Comment by NicolasCARPi** [comment]
Meeting starts in 10: https://zoom.us/j/97692980468?pwd=GJaIVTumFrMkv9UykQlA85hMjPqUxT.1

**Comment by NicolasCARPi** [comment]
Minutes:

- use `additionalType` with value `https://the.elnconsortium.org/eln/1.1` and keep the `conformsTo` with RO-Crate. This way we stay compatible with ro-crate related tools but consumers can have the information that it is a .eln conforming file, where the eln spec is slightly more stringent/precise on some aspects.
- in examples: first dataset should be the producing eln name or something like "This is an eLabFTW experiment", maybe add logo of the software as files, so it becomes easy to show interoperability after import and discriminate from the different datasets
- for variablesMeasured, use machine readable `propertyID` (with `.` to separate nested properties). Use unitCodes if possible.

Feel free to complete, amend, comment.

**Comment by nicobrandt** [comment]
Regarding `variableMeasured`, we also discussed about optionally using `identifier` for specifying term IRIs of individual property values in order to reuse existing schemas, ontologies, etc.
